openedx/adg/lms/webinars/tests/test_helpers.py

```diff
@@ -279,24 +283,102 @@ def test_cancel_all_reminders(msg_id, msg_id_field_name, webinar_registration, m
 
 
 @pytest.mark.django_db
-def test_update_webinar_team_registrations(webinar, mocker):
+@pytest.mark.parametrize('invite_all_platform_users', [True, False])
+def test_get_webinar_invitees_emails(invite_all_platform_users, webinar):
     """
-    Tests `schedule_webinar_reminders` is called to schedule reminders for newly added team members and
-    `cancel_all_reminders` is called for the removed team members.
+    Test that `get_webinar_invitees_emails` correctly returns the invitees' emails of a webinar
     """
-    mock_schedule_webinar_reminders = mocker.patch('openedx.adg.lms.webinars.helpers.schedule_webinar_reminders')
-    mock_cancel_all_reminders = mocker.patch('openedx.adg.lms.webinars.helpers.cancel_all_reminders')
+    UserFactory(email='learner@email.com')
 
     mock_webinar_form = Mock()
     mock_webinar_form.instance = webinar
-    users = UserFactory.create_batch(6)
     mock_webinar_form.cleaned_data = {
-        'co_hosts': users[0:2],
-        'presenter': users[2],
-        'panelists': users[3:]
+        'invites_by_email_address': ['guest1@email.com', 'guest2@email.com'],
```

Comment:
Are we testing the scenario where no invitations are added by email addresses?  Or if invalid/empty emails are added?

Reply:
Emails are validated in the form's clean method, so by this point there are no empty or invalid emails. If you look at the original function that is being tested, it would further clarify.

Reply:
In that case, we can go ahead!